### PR TITLE
doc: Clarify description of `SceneTree.create_timer()`

### PR DESCRIPTION
**Pull request by biswas08433**
Added the 'emit' and the 'signal' word to the 'create_timer()' method description, regarding the timer behavior.

### CODE REVIEW

File: doc/classes/SceneTree.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@
 			<param index="2" name="process_in_physics" type="bool" default="false" />
 			<param index="3" name="ignore_time_scale" type="bool" default="false" />
 			<description>
-				Returns a [SceneTreeTimer] which will [signal SceneTreeTimer.timeout] after the given time in seconds elapsed in this [SceneTree].
+				Returns a [SceneTreeTimer] which will emit the [signal SceneTreeTimer.timeout] signal after the given time in seconds elapsed in this [SceneTree].
```
Comment (AThousandShips) [inline doc/classes/SceneTree.xml:64]:
```suggestion
				Returns a [SceneTreeTimer] which will emit [signal SceneTreeTimer.timeout] after the given time in seconds elapsed in this [SceneTree].
```
Should be sufficient, otherwise too wordy imo

Comment (biswas08433) [reply doc/classes/SceneTree.xml:64]: cool

**Comment by akien-mga** [comment]
Force pushed to do the suggested changes and make the commit message more explicit.

**Review by akien-mga — APPROVED** [review]

**Review by AThousandShips — APPROVED** [review]
